### ISSUE TITLE
USWDS-Site - Bug: Memorable date missing audible announcements of numerals in text fields

### ISSUE THREAD

**Issue by amycole501** [uswds/uswds-site#2392]
### Describe the bug

When testing the memorable date in JAWS on Chrome, I noticed that when I type a numeral(s) into the day and year fields I can't hear what numbers I'm typing in.

### Steps to reproduce the bug

1. Go to https://designsystem.digital.gov/components/memorable-date/
2. Open JAWS in Chrome
3. Type a numeral into the example component.
4. listen for an indication of what numbers you type in. 

### Expected Behavior

On step 4 above, one should hear "one, two, three" or whatever number they typed in being announced. Currently it's silent. The only way to find out what was typed in is to tab out of the field, then back in. Only then does the software tell you what numbers are in the field.

### System setup

PC
Chrome
JAWS

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [X] I checked the [current issues](https://github.com/uswds/uswds-site/issues) for duplicate bug reports.

**Comment by brunerae** [comment]
Hi @amycole501 / @alex-hull would you all test this in the [isolated Storybook Component for Memorable Date](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.sites.pages.cloud.gov/preview/uswds/uswds/develop/iframe.html?p=all&args=&id=components-form-inputs-memorable-date--memorable-date&viewMode=story) so we can determine if its happening just on our site or across the Component?

**Comment by alex-hull** [comment]
Tested with JAWS and NVDA. Both state the numbers that you have entered into the day and year fields. Example, when entering the number ten, you hear, "one zero". When you tab back to the text field, you hear the number as a whole number, "ten". 

One thing I did notice is that each time you entered into a form field, it stated the example given (January 19, 2000). I do not think this is an issue as it provides the user with the format they should be typing in. But wanted to flag it, as it does sound a bit repetitive.
